gitpod-io/gitpod
# add metric to track failed manifest requests from registry-facade

## Description

Add metric to track failed manifest requests.

## Related Issue(s)

Fixes #7886 

## How to test

Run registry facade and block traffic to upstream. Observe metric counter increasing.

## Release Notes

```release-note
 Add metric for tracking failed manifest requests from registry-facade
```

## Documentation
